### PR TITLE
Add linting

### PR DESCRIPTION
**Pull request by hkwu**
Think it would be nice to keep the style consistent throughout the entire repo. Lemme know if the lint rules need adjusting.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "komada-pieces",
+  "description": "This repository contains the various *Pieces* submitted by users and collaborators.",
+  "scripts": {
+    "lint": "eslint commads dataProviders functions inhibitors packages"
```
Comment (CyberiumShadow) [inline package.json]: Typo `commads` should be `commands`

**Review by CyberiumShadow — CHANGES_REQUESTED** [review]
Also I believe Komada Core has its own eslint file which we can use

**Comment by hkwu** [comment]
Yeah, I can include the rules in the core project's config. But the config there is pretty anemic and it isn't enforcing much of a consistent style, which is why I didn't just copy that one.

**Review by CyberiumShadow — APPROVED** [review]
Looks good. I'll merge this now and bring a copy of it over to Komada Core to enforce the linting.

**Comment by CyberiumShadow** [comment]
Holy fuck I didn't realise how many errors this would throw. @hkwu If you can do some linting/beautifying as well to help me out that would be appreciated.

**Comment by hkwu** [comment]
lol, yeah it's going to have tons of errors on first lint. Will do some fixes and adjustments to the rules where needed.

**Comment by CyberiumShadow** [comment]
If you can do that, I'd appreciate it. I'lve already blitzed through the majority of the files on Pieces so far but I'm willing to use updated/optimized rules.

Do note, I am potentially going to use the same eslintrc on Komada Core.

And I''m also using this as well. (just remove the .txt)
[.jsbeautifyrc.txt](https://github.com/eslachance/komada-pieces/files/578776/default.jsbeautifyrc.txt)

**Comment by hkwu** [comment]
You shouldn't need JS Beautifier anymore actually. ESLint can fix most of those automatically for you.

**Comment by CyberiumShadow** [comment]
Righto.

**Comment by CyberiumShadow** [comment]
I'm getting a headache from the Air-BnB linter.

THoughts on this one? https://github.com/CyberiumShadow/selfbot/blob/review/.eslintrc.json

**Pull request merged**